app/actions/remote/entry/app.ts

```diff
@@ -38,14 +39,18 @@ export async function appEntry(serverUrl: string, since = 0, isUpgrade = false)
     if ('error' in entryData) {
         return {error: entryData.error};
     }
-    const {models, initialTeamId, initialChannelId, prefData, teamData, chData, meData} = entryData;
+    const {models, initialTeamId, initialChannelId, prefData, teamData, chData, meData, shouldPopToRoot} = entryData;
     if (isUpgrade && meData?.user) {
         const me = await prepareCommonSystemValues(operator, {currentUserId: meData.user.id});
         if (me?.length) {
             await operator.batchRecords(me);
         }
     }
 
+    if (shouldPopToRoot) {
```

Comment:
This in here at the appEntry remote action is throwing me off, as this is the entry I don't expect this to be here really.

The only place I'm expecting to see this is in the websocket handler that detects the toggle of CRT.